app-backend/tile/src/main/scala/LayerCache.scala

```diff
@@ -221,27 +221,29 @@ object LayerCache extends Config with LazyLogging with KamonTrace {
     toolRunId: UUID,
     subNode: Option[UUID],
     user: User,
-    voidCache: Boolean = false
+    colorRamp: ColorRamp
   ): OptionT[Future, ColorMap] = traceName("LayerCache.toolRunColorMap") {
-    val cacheKey = s"colormap-$toolRunId-${subNode}-${user.id}"
-    if (voidCache) rfCache.delete(cacheKey)
+    val cacheKey = s"colormap-$toolRunId-${subNode}-${user.id}-${colorRamp.getClass.getName}"
     rfCache.cachingOptionT(cacheKey, doCache = cacheConfig.tool.enabled) {
       traceName("LayerCache.toolRunColorMap (no cache)") {
         for {
-          (tool, toolRun) <- LayerCache.toolAndToolRun(toolRunId, user, voidCache)
-          (ast, params) <- LayerCache.toolEvalRequirements(toolRunId, subNode, user, voidCache)
+          (tool, toolRun) <- LayerCache.toolAndToolRun(toolRunId, user)
+          (ast, params) <- LayerCache.toolEvalRequirements(toolRunId, subNode, user)
           nodeId <- OptionT.pure[Future, UUID](subNode.getOrElse(ast.id))
           metadata <- OptionT.fromOption[Future](params.metadata.get(nodeId))
-          cRamp <- OptionT.fromOption[Future](metadata.colorRamp).orElse(OptionT.pure[Future, ColorRamp](geotrellis.raster.render.ColorRamps.Viridis))
-          cmap <- OptionT.fromOption[Future](metadata.classMap.map(_.toColorMap)).orElse({
-            for {
-              breaks <- OptionT.fromOption[Future](metadata.breaks)
-            } yield cRamp.toColorMap(breaks)
-          }).orElse({
-            for {
-              hist <- OptionT.fromOption[Future](metadata.histogram).orElse(LayerCache.modelLayerGlobalHistogram(toolRunId, subNode, user, voidCache))
-            } yield cRamp.toColorMap(hist)
-          })
+          cRamp <- OptionT.fromOption[Future](metadata.colorRamp)
```

Comment:
Doesn't seem like this is necessary any longer -- it's a required argument for the function so no reason to do the options here I think